Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -211,4 +204,11 @@ def git_log_trailers_cmd(repo_path, person, trailer, args)
 
     Utils.safe_popen_read(*cmd).lines.count { |l| l.include?(person) }
   end
+
+  sig { params(repo_full_name: String, person: String, args: Homebrew::CLI::Args).returns(Integer) }
+  def count_reviews(repo_full_name, person, args)
+    GitHub.count_issues("", is: "pr", repo: repo_full_name, reviewed_by: person, review: "approved", args: args)
+  rescue GitHub::API::ValidationFailedError
+    0 # Users who have made their contributions private are not searchable to determine counts.
```

Comment:
Could we output that this might be because they've made their contributions private?

Reply:
As a `--verbose` message or in the data?

Reply:
I think ideally something like an `Error:` if you explicitly request this only this user by name and a `Warning:` if you request multiple users or request them implicitly?